src/pages/Login.tsx

```diff
@@ -1,15 +1,86 @@
-import NavBar from '../components/NavBar';
-const Login = (): JSX.Element => {
-  return (
-    <>
-      <div className="App">
-        <header>
-          <h1>Stock Race</h1>
-          <NavBar />
-          <h2>Login</h2>
-        </header>
-      </div>
-    </>
-  );
-};
-export default Login;
+import React, { useState } from "react";
+import NavBar from "../components/NavBar";
+    
+const Login = (): JSX.Element =>{
+
+    const initialValues = {
+        username: '',
+        password: '',
+    };
+
+    const [values,setValues] = useState(initialValues);
+    const [incorrectPassword, setIncorrectPassword] = React.useState<boolean>(false);
+    
+    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault()
+        const target = e.target as typeof e.target & {
+            username: { value: string };
+            password: { value: string };
+          };
+
+        //if(target correct ) { redirect to logged in home page} 
```

Comment:
These comments won't be needed in development

Reply:
I think I see what are you are doing, this is for the future, right? 

A good  `//Todo: handle a wrong password` is good enough, in my opinion

Reply:
Thanks. Yes it is for the future. For brevity I changed it and re-pushed using your recommendations !